src/station_cmd.cpp

```diff
@@ -3185,13 +3193,17 @@ static void GetTileDesc_Station(TileIndex tile, TileDesc *td)
 
 			if (spec->grf_prop.grffile != nullptr) {
 				const GRFConfig *gc = GetGRFConfig(spec->grf_prop.grffile->grfid);
-				td->grf = gc->GetName();
+				td->station_grf = gc->GetName();
 			}
 		}
 
 		const RailtypeInfo *rti = GetRailTypeInfo(GetRailType(tile));
-		td->rail_speed = rti->max_speed;
-		td->railtype = rti->strings.name;
+		td->rail_desc.speed = rti->max_speed;
+		td->rail_desc.type = rti->strings.name;
+		if (rti->grffile[0] != nullptr) {
+			const GRFConfig* gc = GetGRFConfig(rti->grffile[0]->grfid);
```

Comment:
```suggestion
			const GRFConfig *gc = GetGRFConfig(rti->grffile[0]->grfid);
```
and several others

Reply:
Ahhhg!!!
This is Visual Studio formatting at its own will every time I paste code. Fixing it :-)